Dockerfile

```diff
@@ -30,19 +26,26 @@ COPY environment.yml /tmp/environment.yml
 RUN mamba env create --file /tmp/environment.yml
 
 # Set work directory for xcube installation
-RUN mkdir /xcube
-WORKDIR /xcube
+WORKDIR /home/xcube
 
 # Copy sources into xcube
-COPY . /xcube
+COPY . ./
 
 # Setup xcube package.
 RUN source activate xcube && python setup.py install
 
+WORKDIR /tmp
+ADD scripts/install_xcube-datastore.sh ./
+
+ENV XCUBE_SH_VERSION=0.8.0
+RUN if [[ ${INSTALL_PLUGINS} == '1' ]]; then bash install_xcube-datastore.sh xcube-sh ${XCUBE_SH_VERSION} release; fi;
+ENV XCUBE_CCI_VERSION=0.8.1.dev3
+RUN if [[ ${INSTALL_PLUGINS} == '1' ]]; then bash install_xcube-datastore.sh xcube-cci ${XCUBE_CCI_VERSION} release; fi;
+ENV XCUBE_CDS_VERSION=0.8.1
+RUN if [[ ${INSTALL_PLUGINS} == '1' ]]; then bash install_xcube-datastore.sh xcube-cds ${XCUBE_CDS_VERSION} release; fi;
+
 # Export web server port
 EXPOSE 8080
 
 # Start bash, so we can invoke xcube CLI.
-ENTRYPOINT ["/bin/bash", "-c"]
-# By default, activate xcube environment and print usage help.
-CMD ["source activate xcube && xcube serve -v --address 0.0.0.0 --port 8080 -c ./examples/serve/demo/config.yml"]
+CMD ["/bin/bash"]
```

Comment:
Good!

Reply:
But where is the activation of the xcube env?

Reply:
In this case, the user will have to do it